phoenix_gazebo/webots_project/worlds/purdue_track.wbt

```diff
@@ -9,9 +9,13 @@ EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/project
 EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/trees/protos/SimpleTree.proto"
 EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/trees/protos/Forest.proto"
 EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/traffic/protos/PedestrianCrossing.proto"
-EXTERNPROTO "../protos/Phoenix.proto"
```

Comment:
Pretty sure you need to keep this, since the webots Ros launch file launches a robot called Phoenix. If this externproto is gone then it can't find phnx.